src/Bicep.LangServer/Snippets/Templates/res-firewall-standard.bicep

```diff
@@ -0,0 +1,27 @@
+// Azure Firewall Standard SKU
+resource /*${1:firewall}*/firewall 'Microsoft.Network/azureFirewalls@2021-05-01' = {
+  name: /*${2:'name'}*/'name'
+  location: /*${3:location}*/'location'
+  properties: {
+    sku: {
+      name: 'AZFW_VNet'
+      tier: 'Standard'
+    }    
+    firewallPolicy: {
+      id: /*${4:'firewallPolicy.id'}*/'firewallPolicy.id'

```

Comment:
I'd still prefer to see firewallPolicy.id in the snippet snippet rather than 'firewallPolicy.id', even though (or maybe because) it will show an error message.

@bhsubra / @ucheNkadiCode Do you have a different opinion?  Same for 'subnet.id', 'publicIPAddress.id'